src/System.Net.Primitives/src/System/Net/TransportContext.cs

```diff
@@ -2,12 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Security.Authentication.ExtendedProtection;
 
 namespace System.Net
 {
     public abstract class TransportContext
     {
         public abstract ChannelBinding GetChannelBinding(ChannelBindingKind kind);
+
+        public virtual IEnumerable<TokenBinding> GetTlsTokenBindings()
+        {
+            throw new NotSupportedException();
```

Comment:
Oh, wait, nevermind, I see this is exactly what desktop throws.  NSE is good then.